resources/codegen_inputs/variations.rs

```diff
@@ -0,0 +1,112 @@
+#![parse_module(read_fonts::variation)]
+
+/// The [DeltaSetIndexMap](https://learn.microsoft.com/en-us/typography/opentype/spec/otvarcommonformats#associating-target-items-to-variation-data) table format 0
+table DeltaSetIndexMapFormat0 {
+    /// DeltaSetIndexMap format: set to 0.
+    #[format = 0]
+    format: u8,
+    /// A packed field that describes the compressed representation of 
+    /// delta-set indices. See details below.
+    entry_format: u8,
+    /// The number of mapping entries.
+    map_count: u16,
+    /// The delta-set index mapping data. See details below.
+    #[count(((($entry_format & 0x30) >> 4) + 1) as usize * $map_count as usize)]
```

Comment:
I won't tell rod if you don't tell rod?

more seriously I might prefer if we implement a method on the type that calculates the count, and here we just call that method?

Reply:
Will do!

Reply:
I don't mind the math - would be ok with some sort of "thou shalt implement a fn" too - but the as usize does bug me.